INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 21:51:00] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 21:51:20] "GET /api/v1.0/token HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:12:24] "GET /api/v1.0/token HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:14:10] "GET /api/v1.0/token HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:15:49] "GET /api/v1.0/resource HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:16:14] "GET /api/v1.0/resource HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:20:54] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:21:23] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:42:54] "GET /api/v1.0/resource HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:52:35] "OPTIONS /api/login HTTP/1.1" 404 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:52:38] "OPTIONS /api/login HTTP/1.1" 404 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:52:42] "OPTIONS /api/login HTTP/1.1" 404 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:53:18] "OPTIONS /api/login HTTP/1.1" 404 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:55:17] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 22:55:17] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:06:29] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:06:29] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:11:20] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:11:20] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:16:37] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:16:37] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:16:53] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:16:53] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:17:15] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:17:15] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:17:48] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:17:48] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:21:54] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:21:54] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:22:26] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:22:26] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:28:07] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:28:07] "GET /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:28:50] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:28:50] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:34:45] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:34:45] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:37:46] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:37:46] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:39:03] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [20/Apr/2018 23:39:03] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:05:17] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:05:17] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:06:29] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:06:29] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:12:24] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:12:24] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:16:35] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:16:35] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:16:46] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:16:46] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:21:00] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 00:21:00] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:10:29] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:10:29] "GET /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:21:46] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:21:46] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:22:09] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:22:09] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:22:09] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:22:09] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:42:40] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 09:42:40] "GET /api/v1.0/login HTTP/1.1" 405 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:33:31] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:33:31] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:41:27] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:41:27] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:56:04] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:56:04] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:57:11] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:57:11] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:59:31] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 10:59:31] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:00:30] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:00:30] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:02:33] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:02:33] "POST /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:05:13] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:05:13] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:12:26] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:12:26] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:15] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:15] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:19] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:20] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:21] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:21] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:21] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:21] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:21] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:22] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:22] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:22] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:32] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:13:32] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:30:46] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:30:46] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:32:10] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:32:10] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:34:21] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:34:21] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:34:49] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:34:49] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:39:09] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 11:39:09] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:00:01] "GET /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:00:26] "GET /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:01:49] "GET /api/v1.0/ HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:07:06] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:07:06] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:07:18] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:07:18] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:10:25] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:10:25] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:11:31] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:11:31] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:13:37] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:13:37] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:47:45] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:48:12] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:48:12] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:50:16] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:50:16] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:51:37] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:51:37] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:53:43] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:53:43] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:58:16] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 12:58:16] "GET /api/v1.0/login HTTP/1.1" 401 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 13:41:30] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 13:52:47] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 13:53:43] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 13:54:15] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:00:44] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:02:38] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:07:03] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:10:16] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:11:14] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\models.py', reloading
INFO _internal.py:88  * Restarting with stat
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:44:25] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:44:47] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:49:28] "GET /api/v1.0/eyJhbGciOiJIUzI1NiIsImlhdCI6MTUyNDI4MzIyNiwiZXhwIjoxNTI0MzI2NDI2fQ.eyJpZCI6MX0.fzbjJamKO-Nk9ATN1P4DH4FkU2H0lFXoB50FBDKTz-w HTTP/1.1" 404 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 14:50:37] "GET /api/v1.0/ HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\models.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:10:49] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:10:49] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:12:00] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:12:00] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:14:31] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:14:31] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:16:57] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:16:57] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:17:03] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:17:03] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:17:49] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:17:49] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:19:05] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:19:05] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:22:05] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:22:05] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:22:42] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:22:42] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:23:29] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:23:30] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:23:33] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:25:52] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:25:52] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:26:02] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:26:02] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:26:48] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:26:48] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:28:01] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:28:01] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:28:11] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:28:11] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:29:45] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:29:45] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:31:02] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:31:02] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:31:35] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:31:35] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:35:41] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:35:41] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:43:24] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:43:24] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:44:43] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:44:43] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:45:48] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:45:48] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:56:10] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:56:10] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:56:57] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:56:57] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 15:59:08] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:00:10] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:00:33] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:07:56] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:08:35] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:10:46] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:10:51] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:10:51] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:11:03] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:11:03] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:11:30] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:11:30] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:21:11] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:22:29] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:23:03] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:23:03] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:23:08] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:24:35] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:25:02] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:25:59] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:26:07] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:26:24] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:27:10] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:29:46] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:31:02] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:34:03] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:34:56] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:35:10] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:36:55] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:37:47] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:39:54] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:40:23] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:42:24] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:42:30] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:42:30] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:42:39] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:42:39] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:43:26] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:43:26] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:44:22] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:44:22] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:48:55] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:48:59] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:48:59] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:52:30] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:52:33] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 16:52:33] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 18:43:01] "OPTIONS /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 18:43:01] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 18:43:20] "OPTIONS /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [21/Apr/2018 18:43:20] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:30:03] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:31:34] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\verify.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\config.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:48:19] "POST /api/v1.0/mailcode HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:51:46] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:52:19] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:52:40] "POST /api/v1.0/mailcode HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:52:52] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:56:27] "POST /api/v1.0/mailcode HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:56:37] "POST /api/v1.0/signin HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:57:36] "POST /api/v1.0/signin HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 01:58:55] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 02:01:21] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 02:08:17] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 02:09:25] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 02:11:06] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:46:02] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:46:02] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:50:19] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:50:19] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:56:48] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:56:48] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:58:03] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:58:03] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:58:53] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 08:58:53] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:04:14] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:04:14] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:04:35] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:04:35] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:05:00] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:05:00] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:06:37] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:06:37] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:13:19] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:13:19] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:15:55] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:15:55] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:37:36] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:37:36] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:44:05] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 09:44:05] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 10:13:39] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 10:13:39] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 10:39:34] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 10:39:34] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 10:47:40] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 10:53:07] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:06:52] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:14:26] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:15:53] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:17:03] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:18:33] "POST /api/v1.0/login HTTP/1.1" 500 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:19:26] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:20:14] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:20:14] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:24:03] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:24:03] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:26:58] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:26:58] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:44:33] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:44:33] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:47:14] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:47:14] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:47:17] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:48:31] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:48:31] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:53:48] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:53:48] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:57:19] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:57:19] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:57:21] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:59:17] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 11:59:17] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:46:32] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:46:32] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:47:12] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:47:12] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:48:37] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:48:37] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:48:40] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\config.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:56:41] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:56:41] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:58:01] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 14:58:32] "POST /api/v1.0/login HTTP/1.1" 400 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:00:47] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:01:35] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:02:43] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:03:07] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:03:41] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:03:41] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:04:23] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:04:24] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\passport.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\app\\api_v1\\verify.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\config.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:09:21] "OPTIONS /api/v1.0/mailcode HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:09:21] "POST /api/v1.0/mailcode HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:09:45] "OPTIONS /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:09:45] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:12:11] "OPTIONS /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:12:11] "POST /api/v1.0/signin HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:12:56] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:12:56] "POST /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in 'E:\\WebProject\\MyProject\\flask-user-API-template\\config.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 163-446-745
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:16:42] "OPTIONS /api/v1.0/login HTTP/1.1" 200 -
INFO _internal.py:88 127.0.0.1 - - [22/Apr/2018 15:16:42] "POST /api/v1.0/login HTTP/1.1" 200 -
ERROR __init__.py:98 Error: Directory migrations already exists and is not empty
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:05:04] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:08:08] "[33mGET /signin HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:08:08] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:08:09] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:12:46] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 038393
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:12:47] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:14:28] "[37mOPTIONS /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:14:28] "[37mPOST /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:14:31] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:14:31] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:53:30] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:53:30] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:53:47] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 065150
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:53:47] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:58:58] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 055198
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 15:58:58] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:01:15] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 043269
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:01:15] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/config.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:02:36] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 073009
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:02:36] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
INFO _internal.py:113  * Detected change in '/mnt/f/github/flask-user-api-template/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 305-647-463
ERROR __init__.py:98 Error: Directory migrations already exists and is not empty
ERROR __init__.py:98 Error: Directory migrations already exists and is not empty
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 740-802-018
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:47:27] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:47:27] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 16:47:31] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/passport.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 740-802-018
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 740-802-018
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 17:27:37] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [28/Apr/2020 17:27:38] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/passport.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 740-802-018
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 740-802-018
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/manage.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
DEBUG connectionpool.py:230 Starting new HTTP connection (1): localhost:8080
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[35m[1mGET / HTTP/1.1[0m" 500 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[37mGET /?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[37mGET /?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[37mGET /?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[37mGET /?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:18:26] "[37mGET /?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/manage.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:34:13] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:34:13] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:34:14] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/passport.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:36] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:36] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:37] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:37] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:38] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:38] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:38] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:35:38] "[33mGET / HTTP/1.1[0m" 404 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/passport.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:36:50] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:37:01] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/manage.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:37:07] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:37:07] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:50:13] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:50:14] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:50:15] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:50:15] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:50:15] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:50:15] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/manage.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/manage.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:52:10] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:52:11] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 14:52:16] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/passport.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:15:00] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:15:00] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:15:53] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:15:54] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:16:26] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 072255
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:16:26] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:18:49] "[37mOPTIONS /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:18:49] "[37mPOST /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:18:52] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:18:52] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:19:41] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 037168
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:19:41] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/verify.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:25:14] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:44 邮箱验证码为: 036475
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:25:14] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:44 邮箱验证码为: 001261
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:25:14] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/api_v1/verify.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:26:45] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 008256
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:26:45] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/utils/email.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:27:53] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 013461
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:27:53] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:28:18] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:28:18] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:28:38] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 088552
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:28:38] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:29:16] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 003337
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:29:16] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:29:29] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 082929
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:29:29] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:30:13] "[37mOPTIONS /api/v1.0/signin HTTP/1.1[0m" 200 -
DEBUG passport.py:34 082929
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:30:13] "[37mPOST /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:30:27] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 079332
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:30:27] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:31:14] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 014406
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:31:14] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:31:27] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 048186
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:31:27] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:31:46] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 002006
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:31:46] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:32:00] "[37mOPTIONS /api/v1.0/signin HTTP/1.1[0m" 200 -
DEBUG passport.py:34 002006
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:32:00] "[37mPOST /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:32:22] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 025852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:32:22] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:18] "[37mOPTIONS /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:18] "[37mPOST /api/v1.0/signin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:21] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:21] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:44] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:44] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:56] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:33:56] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:34:12] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:34:12] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:34:16] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:34:32] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 005499
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:34:32] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:35:00] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 014335
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:35:00] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:35:32] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 036151
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:35:32] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:37:17] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 022969
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:37:17] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:37:56] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
DEBUG verify.py:43 邮箱验证码为: 041995
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:37:56] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:58:06] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:58:06] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:58:34] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 16:58:34] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 17:00:20] "[37mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 17:00:20] "[37mPOST /api/v1.0/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 17:00:42] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [30/Apr/2020 17:00:42] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 08:59:16] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 08:59:16] "[31m[1mPOST /api/v1.0/login HTTP/1.1[0m" 405 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 08:59:18] "[31m[1mPOST /api/v1.0/login HTTP/1.1[0m" 405 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 08:59:27] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 08:59:27] "[31m[1mPOST /api/v1.0/login HTTP/1.1[0m" 405 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 09:00:12] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 09:00:12] "[31m[1mPOST /api/v1.0/login HTTP/1.1[0m" 405 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 192-510-852
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 09:01:41] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 09:01:41] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 09:01:47] "[37mOPTIONS /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [01/May/2020 09:01:47] "[37mPOST /api/v1.0/login HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 197-882-632
ERROR __init__.py:98 Error: Directory migrations already exists and is not empty
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:36:46] "[37mGET / HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:36:47] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:40:41] "[33mOPTIONS /api/v1.0/mailcode HTTP/1.1[0m" 404 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/user/verify.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/user/views.py', reloading
INFO _internal.py:113  * Restarting with stat
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:52:37] "[35m[1mPOST /api/user/mailcode HTTP/1.1[0m" 500 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:52:46] "[35m[1mPOST /api/user/mailcode HTTP/1.1[0m" 500 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:53:32] "[31m[1mPOST /api/user/mailcode HTTP/1.1[0m" 400 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:54:05] "[31m[1mPOST /api/user/mailcode HTTP/1.1[0m" 400 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:54:24] "[31m[1mPOST /api/user/mailcode HTTP/1.1[0m" 400 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:54:38] "[31m[1mPOST /api/user/mailcode HTTP/1.1[0m" 400 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:54:47] "[31m[1mPOST /api/user/mailcode HTTP/1.1[0m" 400 -
DEBUG views.py:112 邮箱验证码为: 094061
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:54:54] "[37mPOST /api/user/mailcode HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 11:57:08] "[37mPOST /api/user/sigin HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:04:58] "[37mPOST /api/user/login HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:19 type object 'Community' has no attribute 'id'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:25:08] "[37mGET /api/community/detail?id=2 HTTP/1.1[0m" 200 -
DEBUG views.py:19 type object 'Community' has no attribute 'id'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:25:50] "[37mGET /api/community/detail?id=2 HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:19 type object 'Community' has no attribute 'id'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:26:58] "[37mGET /api/community/detail?id=2 HTTP/1.1[0m" 200 -
DEBUG views.py:19 type object 'Community' has no attribute 'id'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:27:14] "[37mGET /api/community/detail?id=2 HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:27:58] "[37mGET /api/community/detail?id=2 HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:28:14] "[37mGET /api/community/detail HTTP/1.1[0m" 200 -
DEBUG views.py:59 'BaseQuery' object is not callable
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:29:01] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:60 'BaseQuery' object is not callable
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:29:30] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:30:14] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:30:43] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:31:07] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:31:21] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:31:34] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:32:57] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:35:34] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:35:46] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:61 'BaseQuery' object is not callable
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:36:44] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:37:36] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:61 'BaseQuery' object has no attribute 'sum'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:55:00] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 19:56:23] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:61 'BaseQuery' object has no attribute 'sum'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:02:39] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:61 'BaseQuery' object has no attribute 'sum'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:03:09] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:61 'BaseQuery' object has no attribute 'sum'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:03:20] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
DEBUG views.py:61 'BaseQuery' object has no attribute 'sum'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:06:47] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
DEBUG views.py:61 'BaseQuery' object has no attribute 'sum'
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:06:51] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:07:46] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:09:20] "[35m[1mGET /api/community/info HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:09:55] "[37mGET /api/community/info HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/community/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:10:41] "[37mGET /api/community/detail HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:21:13] "[33mGET /api/school/detail HTTP/1.1[0m" 404 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:22:09] "[33mGET /api/school/detail HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:22:10] "[33mGET /api/school/detail HTTP/1.1[0m" 404 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:22:11] "[33mGET /api/school/detail HTTP/1.1[0m" 404 -
INFO _internal.py:113  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:22:17] "[33mGET /api/school/detail HTTP/1.1[0m" 404 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/__init__.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:22:29] "[35m[1mGET /api/school/detail HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/school/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:29:46] "[35m[1mGET /api/school/detail HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/school/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:30:29] "[35m[1mGET /api/school/detail HTTP/1.1[0m" 500 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/school/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:30:48] "[37mGET /api/school/detail HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:31:17] "[37mGET /api/school/detail?id=1 HTTP/1.1[0m" 200 -
INFO _internal.py:113  * Detected change in '/mnt/f/github/Xueyuanlu-back/app/school/views.py', reloading
INFO _internal.py:113  * Restarting with stat
WARNING _internal.py:113  * Debugger is active!
INFO _internal.py:113  * Debugger PIN: 129-213-639
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:34:23] "[37mGET /api/school/info HTTP/1.1[0m" 200 -
INFO _internal.py:113 127.0.0.1 - - [06/May/2020 20:35:51] "[37mGET /api/school/info HTTP/1.1[0m" 200 -
ERROR app.py:1891 Exception on /api/school/detail [GET]
Traceback (most recent call last):
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/root/XueyuanluDC/back/app/__init__.py", line 65, in catch_all
    # @app.route('/<path:path>')
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
ERROR app.py:1891 Exception on /api/school/detail [GET]
Traceback (most recent call last):
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/root/XueyuanluDC/back/app/__init__.py", line 65, in catch_all
    # @app.route('/<path:path>')
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/jinja2/loaders.py", line 115, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "/root/XueyuanluDC/back/venv/lib/python3.8/site-packages/flask/templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
